uPortal-api/uPortal-api-rest/src/main/java/org/apereo/portal/rest/PagsRESTController.java

```diff
@@ -170,48 +170,49 @@ public class PagsRESTController {
             produces = MediaType.APPLICATION_JSON_VALUE,
             method = RequestMethod.PUT)
     public @ResponseBody String updatePagsGroup(
-            HttpServletRequest req,
-            HttpServletResponse res,
+            HttpServletRequest httpServletRequest,
+            HttpServletResponse httpServletResponse,
```

Comment:
I'm fine with expanding the name a bit, abbreviations are less than ideal.
I'm not sure if we want to match the variable name to its type though (https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat)

Reply:
I believe that the name of a variable plays important role in
describing the behavior to the developer. Unclear, meaningless or confusing names
make the things hard to understand.
Hence, I went ahead and applied this refactoring (SonarQube suggested this smell as well).
Below is the additional link i referred:
https://refactoring.guru/rename-method

Reply:
`request` and `response` would be just as clear without tying it to the type.

Reply:
As advised, I have made this change for these two variables.